src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/FewReadsWriter.java

```diff
@@ -0,0 +1,78 @@
+package org.broadinstitute.hellbender.dev.pipelines.bqsr;
+
+
+import com.google.api.services.genomics.model.Read;
+import com.google.cloud.dataflow.sdk.Pipeline;
+import com.google.cloud.dataflow.sdk.transforms.Create;
+import com.google.cloud.dataflow.sdk.transforms.DoFn;
+import com.google.cloud.dataflow.sdk.transforms.ParDo;
+import com.google.cloud.dataflow.sdk.transforms.View;
+import com.google.cloud.dataflow.sdk.values.PCollection;
+import com.google.cloud.dataflow.sdk.values.PCollectionView;
+import com.google.cloud.genomics.gatk.common.GenomicsConverter;
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMFileWriter;
+import htsjdk.samtools.SAMFileWriterFactory;
+import htsjdk.samtools.SAMRecord;
+import org.broadinstitute.hellbender.utils.dataflow.BucketUtils;
+
+import java.io.OutputStream;
+import java.io.Serializable;
+
+/**
+ * Takes a few Reads and will write them to a BAM file.
+ * However all the reads must fit into a single worker's memory, so this won't go well if you have too many.
+ */
+public class FewReadsWriter implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Takes a few Reads and will write them to a BAM file.
+     * However all the reads must fit into a single worker's memory, so this won't go well if you have too many.
+     */
+    public static void WriteToFile(Pipeline pipeline, PCollection<Read> reads, final SAMFileHeader header, final String filename) {
+        new FewReadsWriter().inner_WriteToFile(pipeline, reads, header, filename);
+    }
```

Comment:
This seems pointless -- can we get rid of this wrapper method and make `inner_WriteToFile()` itself `public static` (renaming it to `writeToFile()`)? This would also require making the `SaveToFile` inner class `static`.